docs/plugins/pytorch_plugin_api.rst

```diff
@@ -7,3 +7,12 @@ PyTorch Plugin API reference
    :inherited-members:
    :exclude-members: TorchPythonFunction
 
+
+.. autoclass:: nvidia.dali.plugin.pytorch.experimental.proxy.DALIServer
+   :members:
+   :special-members: __init__, __enter__, __exit__
+
+
+.. autoclass:: nvidia.dali.plugin.pytorch.experimental.proxy.DataLoader
+   :members:
+   :special-members: __getitem__, __init__
```

Comment:
I would consider placing them in the same page as the rest od DALI Proxy documentation or at leas separate them with some subsection.

Reply:
moved